pquery3-run.conf

```diff
@@ -0,0 +1,256 @@
+#!/bin/bash
+# Created by Mohit Joshi, Percona LLC
+
+################################################################################
+# Welcome to the pquery v3.0 configuration file (pquery3-run.conf). Feel free  #
+# to copy this file, edit the copy & change the CONFIGURATION_FILE variable    #
+# in pquery-run.sh to match!                                                   #
+# Please do not change this file unless your name is listed above. As a        #
+# template/default, it is the only configuration file for pquery v3.0 which    #
+# is maintained whenever pquery-run.sh is updated.                             #
+################################################################################
+
+# ################### User configurable variables: generics 1 ##################
+# pquery3-ps: Percona Server, pquery3-ms: MySQL                                #
+################################################################################
+PQUERY_BIN=$HOME/pquery_n/src/pquery3-ps
```

Comment:
Please note this is incorrect at the moment. Once we have a new repo for pquery3, we can decide upon the path. Also this path can vary from user to user